roles/filesystem/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+---
+- name: Create default share type
+  become: true
+  command: >
+    docker exec kolla_toolbox openstack
+    --os-interface internal
+    --os-auth-url {{ openstack_auth.auth_url }}
+    --os-identity-api-version 3
+    --os-project-domain-name {{ openstack_auth.user_domain_name }}
+    --os-project-name {{ keystone_admin_project }}
+    --os-username {{ keystone_admin_user }}
+    --os-password {{ keystone_admin_password }}
+    --os-user-domain-name {{ openstack_auth.user_domain_name }}
+    --os-region-name {{ openstack_region_name }}
+    share type create {{ default_share_type }} false
+    --extra-specs vendor_name=Ceph storage_protocol=NFS
```

Comment:
I see that openstack.cloud doesn't have a module for this. I think this is OK for now, but we should consider either making our own module, or contributing to upstream.

Reply:
I'll have it as it is for now. will keep in mind updating this in the future.